pkgs/by-name/sl/slicer3d/package.nix

```diff
@@ -0,0 +1,133 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+  hwloc,
+  fetchurl,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
+
+  hwloc_old = hwloc.overrideAttrs (
+    old:
+    let
+      hwloc_version = "1.11.9";
+    in
+    {
+      version = hwloc_version;
+      src = fetchurl {
+        url = "https://www.open-mpi.org/software/hwloc/v${lib.versions.majorMinor hwloc_version}/downloads/hwloc-${hwloc_version}.tar.bz2";
+        sha256 = "sha256-OUMzGEJI1jyycIqXblfwUzfQO7UMM6owl/9cWnSoUWQ=";
+      };
+    }
+  );
+
+  desktopItem = makeDesktopItem {
+    name = "Slicer3D";
```

Comment:
For clarification this application is not named “Slicer 3D”. The name is “3D Slicer”. 

- https://slicer.readthedocs.io/en/latest/user_guide/about.html#the-3d-slicer-name-and-logo
- https://github.com/Slicer/Slicer/blob/main/README.md

Reply:
Somewhat confusingly, the `name` attribute is actually the *filename* (as in this desktop file would be called `Slicer3D.desktop`), and the actual human-readable name is under `desktopName`

Reply:
I have somehow used this program for like a year and have thought it was slicer 3d this whole time haha
I'll update that after i finish work today